tests/pointCalculatorTest.js

```diff
@@ -0,0 +1,15 @@
+// link this file to index.html
+
+import { calculatePoints } from '../src/app/logic/calculatePoints.js';
+
+const pointsMediumTrue = calculatePoints(10, true, 'medium');
```

Comment:
You should try to start writing unit tests with jest, these cases would be really simple to write and would give you basic testing experience

Reply:
in next project! necessarily!